pages/docs/react/latest/components-and-props.mdx

```diff
@@ -328,40 +318,42 @@ var make = App;
 
 ### Handwritten Components
 
-You don't need to use the `@react.component` decorator to write components that can be used in JSX. Instead you can write a pair of `make` and `makeProps` functions such that type `makeProps: 'a => props` and `make: props => React.element` and these will always work as React components.
+You don't need to use the `@react.component` decorator to write components that can be used in JSX. Instead you can write the `make` function with type `props` and these will always work as React components.
```

Comment:
But then you will have the issue with the component name being "make" in the React dev tools.

Reply:
Thanks! Added https://github.com/rescript-association/rescript-lang.org/pull/593/commits/a2c7b6d243fd8ee8bdeb3ec1aaabef4ac35e3b9c